### PR TITLE
Scrape data command

### PR DESCRIPTION
**Pull request by maxdg99**
- Integrates all of our scraping efforts into one scrape command: `python manage.py scrape`.
- Moves over to a name.# schema for instructor ID

### CODE REVIEW

File: sei_website/sei_website_app/management/scrape/scrape_post_2018.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
+import requests as req
+import pandas as pd
+import json
+from sei_website_app.management.scrape.base_scraper import DEPARTMENT_CODE, DEPARTMENT_NAME, BaseScraper
+from os.path import exists
+import re
+from sei_website_app.models import Department
+
+USER_ID = "tzIsE9d1Yhs1zr3yqg4mmuJGbBZUgIuxAwZgaTk7ktRmJZNFp8naKp"
```
Comment (adambricelis) [inline sei_website/sei_website_app/management/scrape/scrape_post_2018.py:9]: We should put this in the .env file

File: README.md [hunk 2]
```diff
@@ -55,6 +55,12 @@ Website which makes publicly-available instructor rating data more accessible an
     1. [Set the domain name in the Django admin](https://stackoverflow.com/questions/15402622/django-sitemap-http-example-com)
     1. Populate the database
 
+## Scraping Data
```
Comment (adambricelis) [inline README.md:58]: If we add USER_ID to the .env file, we need to add that to these instructions

**Review by adambricelis — CHANGES_REQUESTED** [review]
Nice work! Thanks for deleting all of those files we don't need anymore.

File: README.md (hunk 2 above)
Comment (maxdg99) [reply README.md:64]: Done

File: sei_website/sei_website_app/management/scrape/scrape_post_2018.py (hunk 1 above)
Comment (maxdg99) [reply sei_website/sei_website_app/management/scrape/scrape_post_2018.py]: Done

**Pull request merged**